packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -1221,6 +1221,33 @@ void main() {
       );
     });
 
+    testWidgets('DatePicker displays the date in correct order', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: Center(
+            child: SizedBox(
+              height: 400.0,
+              width: 400.0,
+              child: CupertinoDatePicker(
+                dateOrder: DatePickerDateOrder.ydm,
+                mode: CupertinoDatePickerMode.date,
+                onDateTimeChanged: (DateTime newDate) {},
+                initialDateTime: DateTime(2018, 1, 14, 10, 30),
+              ),
+            ),
+          ),
+        ),
+      );
+
+      Offset lastOffset = tester.getTopLeft(find.text('2018'));
+
+      expect(tester.getTopLeft(find.text('14')).dx > lastOffset.dx, true);
+      lastOffset = tester.getTopLeft(find.text('14'));
+
+      expect(tester.getTopLeft(find.text('January')).dx > lastOffset.dx, true);
+      lastOffset = tester.getTopLeft(find.text('January'));
```

Comment:
Is this last line required? I mean the assignment is not being used.

Maybe you meant to remove it or add another `expect` statement.